test/externalTests/solc-js/solc-js.sh

```diff
@@ -23,41 +16,44 @@
 # You should have received a copy of the GNU General Public License
 # along with solidity.  If not, see <http://www.gnu.org/licenses/>
 #
-# (c) 2017 solidity contributors.
+# (c) 2019 solidity contributors.
 #------------------------------------------------------------------------------
+source scripts/common.sh
+source test/externalTests/common.sh
 
-set -e
-
-if [ ! -f "$1" -o -z "$2" ]
-then
-  echo "Usage: $0 <path to soljson.js> <version>"
-  exit 1
-fi
-
+verify_input "$1" "$2"
+# verify_input "$2"
 SOLJSON="$1"
 VERSION="$2"
 
-DIR=$(mktemp -d)
-(
-    echo "Preparing solc-js (master_060)..."
-    git clone --depth 1 --branch master_060 https://github.com/ethereum/solc-js "$DIR"
-    cd "$DIR"
-    # disable "prepublish" script which downloads the latest version
-    # (we will replace it anyway and it is often incorrectly cached
-    # on travis)
-    npm config set script.prepublish ''
-    npm install
-
-    # Replace soljson with current build
-    echo "Replacing soljson.js"
-    rm -f soljson.js
-    cp "$SOLJSON" soljson.js
+function install_fn { echo "Nothing to install."; }
```

Comment:
Doesn't it need `npm intsall` for `solc-js`?

Reply:
Or it is part of `setup`?

Reply:
Yes, it's part of `setup_solcjs`. Here we don't need these additional steps since we're only downloading solcjs and not as we do for the other external tests solcjs + the ext. repo.